app/views/shared/_media_item.html.haml

```diff
@@ -1,12 +1,12 @@
--if commercial_type == CONFIG['commercial_types']['slideshare']
+-if commercial_type == Commercial.provider['slideshare']
   %iframe{:width=>"560", :height=>"315", :frameborder=>"0", :allowfullscreen=>"true", :src=> (request.ssl? ? "https://" : "http://") + "www.slideshare.net/slideshow/embed_code/#{commercial_id}"}
```

Comment:
:width=>"100%"
:height=>"100%"

should work well with a [responsive embed](http://getbootstrap.com/components/#responsive-embed)